### PR TITLE
CLI update for wdio-electron-service v5

### PR DESCRIPTION
**Pull request by goosewobbler**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

`wdio-electron-service` v5 has recently been [released](https://github.com/webdriverio-community/wdio-electron-service/releases/tag/5.0.0) which significantly changed the interface with WDIO.  The `wdio-cli` package no longer produces valid configuration for the service so needs to be updated.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

Electron Forge is explicitly mentioned here but is not supported by the service `getBinaryPath` util yet.  All non-`electron-builder` cases fall back to asking for the appBinaryPath, as does the case where the `electron-builder` config file is not JSON.

### CODE REVIEW

**Review by christian-bromann — APPROVED** [review]
This is awesome, thanks!

🙏

**Comment by goosewobbler** [comment]
@christian-bromann I simplified this a fair bit for the electron-builder case.  Basically it just asks for the EB config path and if a JSON file, pulls out the relevant data and passes it to the `getBinaryPath` util.  Other types of EB config file are treated the same as the non-EB cases, e.g. fall back to asking for the electron binary path.

I couldn't find a way to easily test the output in units so just did some manual verification...

**Comment by christian-bromann** [comment]
> I couldn't find a way to easily test the output in units so just did some manual verification...

I tested myself and looks good to me. There is an open issue for testing the CLI wizard behavior and it is a more complex process.

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

File: packages/wdio-cli/src/types.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+import type { BackendChoice, RegionOptions, CompilerOptions, ElectronBuildToolChoice } from './constants.js'
 
 export interface Questionnair {
     runner: string
     preset?: string
     installTestingLibrary?: boolean
-    appPath?: string
+    electronAppBinaryPath?: string
+    electronBuildTool?: ElectronBuildToolChoice
+    electronBuilderConfigPath: string
```
Comment (jan-molak) [inline packages/wdio-cli/src/types.ts:11]: It seems like this property should have been marked as optional

**Pull request merged**